app/assets/app.js

```diff
@@ -76,7 +76,7 @@ window.addEventListener('DOMContentLoaded', function () {
 
         // open a dialog with the result if found
         if (resultData !== false) {
-            navigator.vibrate(200);
+            //navigator.vibrate(200);
```

Comment:
Maybe you can do something like

```js
navigator.vibrate && navigator.vibrate(200);
```

This way it will still be compatible with compatible browsers ;)